docs/standard-library/openapi3/overview.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
![image](https://github.com/microsoft/typespec/assets/1031227/940a3d3d-78fa-4693-a066-4ed3dd3f7e4e)
In the structure above `overview.md` and `reference/index.md` are pretty much empty with just a reference to the emitter usage.

Would be good to rethink how to want library docs to be structured. Knowing that there can be auto generated ref docs as well as some manually written guides